### PR TITLE
create a reminder for a booked appointment

### PR DESCRIPTION
**Pull request by dyrkin**

### CODE REVIEW

**Comment by atais** [comment]
you're not getting text messages from luxmed?

File: server/src/main/scala/com/lbs/server/lang/Pl.scala [hunk 1]
```diff
@@ -379,4 +379,17 @@ object Pl extends Lang {
   override def canNotDetectPayer(error: String): String = s"Nie udało się ustalić płatnika. Powód: $error"
 
   override def pleaseChoosePayer: String = "<b>➡</b> Nie udało się ustalić domyślnego płatnika, wybierz jakiegoś."
+
+  override def youHaveAppointmentAt(reminder: Reminder): String =
+    s"""👍 Macie wizytę o ⏱ <b>${formatTime(reminder.appointmentTime.toLocalTime)}</b>!
+       |
+       |${capitalize(doctor)}: ${reminder.doctorName}
```
Comment (atais) [inline server/src/main/scala/com/lbs/server/lang/Pl.scala:386]: externalize common parts?

**Comment by dyrkin** [comment]
> 



> you're not getting text messages from luxmed?

I'm getting but a day prior. I just wanted to be notified an hour or two before the event to drive to a facility with no rush.

File: server/src/main/scala/com/lbs/server/lang/Pl.scala (hunk 1 above)
Comment (dyrkin) [reply server/src/main/scala/com/lbs/server/lang/Pl.scala:386]: yeah, i agree. but have no time to do this now. but it's definitely a good idea.

**Comment by atais** [comment]
I see. However I am not sure if the bot should be the one to create such reminders. In the end its fine (and configurable) but to me it seems calendar is the tool of choice for such purpose.

Like with google calendar you can set an appointment with custom notification times:
Screenshot 2022-08-17 at 11 19 02

should be fine with Apple Calendar as well, but I am not using that one currently.

**Comment by atais** [comment]
Maybe this can help 
https://github.com/ical4j/ical4j

**Pull request closed without merging**